File: LAPORAN PRATIKUM/MODUL4/CODEBLOKS DAN PYTHON/SOAL 5/PRAK305-2210817210014-MuhammadQoharyRawidafany.py
for ulangi in range (0,5):
    I = int (input(""))
    H = I / 84600
    J = I / 3600 % 24
    M = I / 60 % 60
    D = I % 60
    if I < 3600 :
        print("{:02d}:{:02d}:{:02d}".format(int(J),int(M),int(D)))
    elif I >= 3600 and I < 84600:
        print("{:02d}:{:02d}:{:02d}".format(int(J),int(M),int(D)))
    elif I >= 84600 :
         print("{} hari {:02d}:{:02d}:{:02d}".format(int(H),int(J),int(M),int(D)))

File: LAPORAN PRATIKUM/MODUL4/CODEBLOKS DAN PYTHON/SOAL 5/PRAK305-2210817210014-MuhammadQoharyRawidafany.c
#include <stdio.h>
int main ()
{
    for (int q=1;q<=5;q++){
    int H,J,D,I,M;
    scanf("%d",&I);
    H = I /86400;
    J = (I / 3600) % 24;
    M = (I / 60) % 60;
    D = I % 60;
    if (I<=3600){
        printf("%02d : %02d : %02d\n",J,M,D);
        }
    else if(I>3600 && I<86400){
        printf("%02d : %02d : %02d\n",J,M,D);
        }
    else if(I>=86400){
        printf("%d hari %02d : %02d : %02d\n",H,J,M,D);
    }
    }
}